bin/preload.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-JARFILE="build/libs/susi_server-all.jar"
+JARFILE="build/libs/susi_server.jar"
```

Comment:
No, I don't think this is a good idea. We later need to deploy the stuff and thus the `-all` variant needs to be build. The correct incantation is `gradle assemble`.

I think it should be only documented that `gradle assemble` is necessary.

Reply:
aah ok, Thankyou @norbusan 
closed the PR